diff --git a/lib/rucio/core/oidc.py b/lib/rucio/core/oidc.py
index c00388e84..80f3fadb5 100644
--- a/lib/rucio/core/oidc.py
+++ b/lib/rucio/core/oidc.py
@@ -64,6 +64,7 @@ IDPSECRETS = config_get('oidc', 'idpsecrets', False)
 ADMIN_ISSUER_ID = config_get('oidc', 'admin_issuer', False)
 EXPECTED_OIDC_AUDIENCE = config_get('oidc', 'expected_audience', False, 'rucio')
 EXPECTED_OIDC_SCOPE = config_get('oidc', 'expected_scope', False, 'openid profile')
+EXTERNAL_EXPECTED_OIDC_SCOPE = config_get('oidc', 'external_expected_scope', False)
 EXCHANGE_GRANT_TYPE = config_get('oidc', 'exchange_grant_type', False, 'urn:ietf:params:oauth:grant-type:token-exchange')
 REFRESH_LIFETIME_H = config_get_int('oidc', 'default_jwt_refresh_lifetime', False, 96)
 
@@ -116,6 +117,9 @@ def _token_cache_set(key: str, value: str) -> None:
     """Store a token in the cache."""
     REGION.set(key, value)
 
+CC_CLIENT_ID = None
+CC_CLIENT_SECRET = None
+
 
 def request_token(audience: str, scope: str, use_cache: bool = True) -> Optional[str]:
     """Request a token from the provider.
@@ -123,6 +127,27 @@ def request_token(audience: str, scope: str, use_cache: bool = True) -> Optional
     Return ``None`` if the configuration was not loaded properly or the request
     was unsuccessful.
     """
+    global CC_CLIENT_ID, CC_CLIENT_SECRET
+
+    # Load SCIM credentials once
+    if not CC_CLIENT_ID or not CC_CLIENT_SECRET:
+        try:
+            with open(IDPSECRETS) as f:
+                data = json.load(f)
+                issuer_data = data.get(ADMIN_ISSUER_ID)
+                if not issuer_data:
+                    logging.error('Issuer "%s" not found in idpsecrets', ADMIN_ISSUER_ID)
+                    return None
+                scim_data = issuer_data.get("SCIM")
+                if not scim_data or not scim_data.get("client_id") or not scim_data.get("client_secret"):
+                    logging.error('SCIM credentials missing for issuer "%s"', ADMIN_ISSUER_ID)
+                    return None
+                CC_CLIENT_ID = scim_data["client_id"]
+                CC_CLIENT_SECRET = scim_data["client_secret"]
+        except Exception:
+            logging.error('Failed to read or parse idpsecrets file "%s"', IDPSECRETS, exc_info=True)
+            return None
+
     if not all([OIDC_CLIENT_ID, OIDC_CLIENT_SECRET, OIDC_PROVIDER_ENDPOINT]):
         if OIDC_CONFIGURATION_RUN or not __load_oidc_configuration():
             return None
@@ -134,7 +159,7 @@ def request_token(audience: str, scope: str, use_cache: bool = True) -> Optional
 
     try:
         response = requests.post(url=OIDC_PROVIDER_ENDPOINT,
-                                 auth=(OIDC_CLIENT_ID, OIDC_CLIENT_SECRET),
+                                 auth=(CC_CLIENT_ID, CC_CLIENT_SECRET),
                                  data={'grant_type': 'client_credentials',
                                        'audience': audience,
                                        'scope': scope})
@@ -982,7 +1007,7 @@ def validate_jwt(json_web_token: str, *, session: "Session") -> dict[str, Any]:
         METRICS.counter(name='JSONWebToken.valid').inc()
         # if token is valid and coming from known issuer --> check aud and scope and save it if unknown
         if token_dict['authz_scope'] and token_dict['audience']:
-            if all_oidc_req_claims_present(token_dict['authz_scope'], token_dict['audience'], EXPECTED_OIDC_SCOPE, EXPECTED_OIDC_AUDIENCE):
+            if all_oidc_req_claims_present(token_dict['authz_scope'], token_dict['audience'], EXTERNAL_EXPECTED_OIDC_SCOPE, EXPECTED_OIDC_AUDIENCE):
                 # save the token in Rucio DB giving the permission to use it for Rucio operations
                 __save_validated_token(json_web_token, token_dict, session=session)
             else:
diff --git a/lib/rucio/transfertool/fts3.py b/lib/rucio/transfertool/fts3.py
index 648470d4e..dd87176cf 100644
--- a/lib/rucio/transfertool/fts3.py
+++ b/lib/rucio/transfertool/fts3.py
@@ -1082,6 +1082,9 @@ class FTS3Transfertool(Transfertool):
             expected_transfer_id = self.__get_deterministic_id(job_params["sid"])
             self.logger(logging.DEBUG, "Submit bulk transfers in deterministic mode, sid %s, expected transfer id: %s", job_params["sid"], expected_transfer_id)
 
+        if self.token and config_get_bool('conveyor', 'fts_unmanaged_token', raise_exception=False, default=False):
+            job_params['unmanaged_tokens'] = True  # type: ignore
+
         # bulk submission
         params_dict = {'files': files, 'params': job_params}
         params_str = json.dumps(params_dict, cls=APIEncoder)
